descartes_core/CMakeLists.txt

```diff
@@ -3,13 +3,12 @@ project(descartes_core)
 
 add_definitions(-std=gnu++0x)
 find_package(catkin REQUIRED COMPONENTS
-  console_bridge
   moveit_core
   roscpp
+  cmake_modules
 )
 
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
```

Comment:
The Eigen library is used in this package.  Wouldn't **find(catkin REQUIRED COMPONENTS cmake_modudes ...)** include Eigen as well;  in which case calling find_package(Eigen ...) would be redundant.  If this is so then adding Eigen_INCLUDE_DIRS or Eigen under DEPENDS isn't required either.